test/sharness/t0130-multinode.sh

```diff
@@ -53,6 +53,18 @@ check_file_fetch() {
 	'
 }
 
+test_resolve() {
+	_nid="$1"; shift
```

Comment:
i would opt for `$1`, `$2`, `$3`. I think thats cleaner than shifting just to be able to use the same variable.

@chriscool what do you think?